data/json/item_groups.json

```diff
@@ -2747,6 +2753,8 @@
       ["mosin91_30_ebr", 1],
       ["mosin44_ebr", 1],
       ["sks", 6],
+      ["m1903", 5],
```

Comment:
Neither of these is going to be common in a modern rifle collection, dial the weight down to 1 or so, probably the same for all the instances of them.